### PR TITLE
[feat] create Homepage

### PR DESCRIPTION
**Pull request by ganesh-kanchi**

### CODE REVIEW

**Review by Avin008 — APPROVED** [review]
looks good to merge

File: src/pages/Homepage/Homepage.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.homepage-content{
+    display: flex;
+    gap: 2rem;
+    justify-content: space-around;
+    align-items: center;
+    width: 80vw;
+    height: auto;
+    position:static;
```
Comment (tusharkandpal) [inline src/pages/Homepage/Homepage.css:8]: Default position is `static` so no need to write it

File: src/pages/Homepage/Homepage.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+export const Homepage = () => {
+    return (
+        <div className="page-container">
+            <Navbar />
+            <main className="page-content homepage-content">
+                <div className="homepage-text">
+                    <div className="heading-1">An easy to use pomodoro for an effective human.</div>
+                    <div className="homepage-links">
+                        <a className="button" href="http://https://medium.com/keep-productive/how-to-use-a-pomodoro-timer-f9d172132981" target="_blank" rel="noopener noreferrer">What is Pomodoro technique?</a>
```
Comment (tusharkandpal) [inline src/pages/Homepage/Homepage.jsx]: The `href` attribute value to medium article is wrong.

**Review by tusharkandpal — APPROVED** [review]
Few suggestions, else good to merge 👍

File: src/pages/Homepage/Homepage.css (hunk 1 above)
Comment (ganesh-kanchi) [reply src/pages/Homepage/Homepage.css:8]: i removed it in later branches

File: src/pages/Homepage/Homepage.jsx (hunk 2 above)
Comment (ganesh-kanchi) [reply src/pages/Homepage/Homepage.jsx]: have fixed it, thank you for pointing out

**Pull request merged**